app/src/main/java/by/paranoidandroid/notesapp/activities/MainActivity.java

```diff
@@ -28,27 +30,29 @@ protected void onCreate(Bundle savedInstanceState) {
         recyclerView.setLayoutManager(new LinearLayoutManager(this));
         recyclerView.addItemDecoration(new DividerItemDecoration(this, DP_MARGIN));
 
-        adapter = new NotesAdapter(this);
+        adapter = new NotesAdapter(this, this);
```

Comment:
Not readable, I suggest to rename them

Reply:
Not readable is namely "NotesAdapter" name, should this adapter be renamed? Should it be more verbose?